src/ApiService/ApiService/onefuzzlib/VmOperations.cs

```diff
@@ -218,7 +218,9 @@ await _context.Creds.GetResourceGroupResource()
                 extensionName,
                 extension
             );
-        } catch (RequestFailedException ex) when (ex.Status == 409 && ex.Message.Contains("VM is marked for deletion")) {
+        } catch (RequestFailedException ex) when
+            (ex.Status == 409 &&
+                (ex.Message.Contains("VM is marked for deletion") || ex.Message.Contains("The request failed due to conflict with a concurrent request."))) {
             _logTracer.Info(ex.Message);
```

Comment:
```suggestion
            _logTracer.Info($"Tried to create extension but failed due to {ex.Message}");
```

So we know what the message is about